keylime/ca_util.py

```diff
@@ -295,6 +300,7 @@ def get_crl_distpoint(cert_path):
     logger.info("No CRL distribution points in %s", cert_path)
```

Comment:
there is something weird here: the extra "pass" and the indentation for sure. But maybe the code scanning complain makes sense too

Reply:
fixed